plugins/woocommerce-blocks/tests/e2e/bin/scripts/parallel/languages.sh

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env bash
-
-wp language core install nl_NL
-wp language plugin install woocommerce nl_NL
```

Comment:
Maybe we're missing something, there's[ CI jobs](https://github.com/woocommerce/woocommerce/actions/runs/11721011507/job/33615015804?pr=52188) complaining about this language not being installed 🤔